lib/components/ConfirmationDialog.jsx

```diff
@@ -34,42 +34,59 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* eslint-disable react/jsx-no-target-blank */
```

Comment:
I think the "right" way is to use `target='_blank' rel='noopener noreferrer'`, as per the documentation at https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

Reply:
OK, I see. I add this since I didn't know how to fix. Now I know it. Thanks! 😄